src/main/java/ch/njol/skript/expressions/ExprSpawnEggEntity.java

```diff
@@ -0,0 +1,123 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.bukkitutil.ItemUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.slot.Slot;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntitySnapshot;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SpawnEggMeta;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Spawn Egg Entity")
+@Description({
+	"Gets or sets the entity snapshot that the provided spawn eggs will spawn when used.",
+	"NOTE: When using an entity data i.e 'a zombie', it will spawn a new zombie to retrieve an entity snapshot and then be removed."
```

Comment:
```suggestion
	"NOTE: When using an entity data (i.e 'a zombie'), it will spawn a new zombie to retrieve an entity snapshot and then be removed."
```